### DISCUSSION TITLE
openvino as pytorch backend - model cache directory

### DISCUSSION THREAD

**Discussion by js333031** [openvinotoolkit/openvino#24008] [Q&A]
I have a jupyter notebook and a simple pytorch test application - attached. I'm specifying the model cache directory as below but no files are created.  The 
The directory gets created but it's empty.  Where are the cached files? I checked after running the whole notebook.
In case the cache is somewhere else, why is the directory created?

`model = torch.compile(model, backend="openvino", options = {"device" : "GPU.1", "model_caching" : True, "cache_dir": "./model_cache"})`

After running the notebook, file statistics:
Size of directories/files:
```
student@nuc21:~/openvino_env/pytorch_test$ du --max-depth=1 -h
8.0K    ./model_cache
12K     ./.ipynb_checkpoints
48K     .
student@nuc21:~/openvino_env/pytorch_test$
```

Files:
```
student@nuc21:~/openvino_env/pytorch_test$ find . -name '*'
.
./pytorch_test.ipynb
./model_cache
./model_cache/model
./.ipynb_checkpoints
./.ipynb_checkpoints/pytorch_test-checkpoint.ipynb
./.ipynb_checkpoints/Untitled-checkpoint.ipynb
student@nuc21:~/openvino_env/pytorch_test$
```
[pytorch_test.md](https://github.com/openvinotoolkit/openvino/files/14960438/pytorch_test.md)

**Comment by ilya-lavrenov** [comment]
CC @suryasidd

**Reply by js333031** [reply]
Looking for updates. Thanks

**Reply by js333031** [reply]
Please provide an update
